CharlieBackend.Api/Middlewares/IsAccountActiveMiddleware.cs

```diff
@@ -27,7 +29,7 @@ public IsAccountActiveMiddleware(RequestDelegate next)
         /// <summary>
         /// 
         /// </summary>
-        public async Task InvokeAsync(HttpContext context, IAccountService accountService)
+        public async Task InvokeAsync(HttpContext context, IAccountService accountService, ICurrentUserService currentUserService)
```

Comment:
Since you are refactoring the class, I think it would be a good idea to add summary inside empty `<summary>` block. At least a description of the purpose of this middleware on the class level.

Reply:
Continued here: https://github.com/ita-social-projects/WhatBackend/pull/372#discussion_r594386946